proposals/eligibility_validator.go

```diff
@@ -116,7 +116,7 @@ func (v *Validator) CheckEligibility(ctx context.Context, ballot *types.Ballot,
 			return fmt.Errorf(
 				"%w: proof contains incorrect VRF signature. beacon: %v, epoch: %v, counter: %v, vrfSig: %s",
 				fetch.ErrIgnore,
-				data.Beacon.ShortString(),
+				data.Beacon,
```

Comment:
`ShortString` used to only print the first 5 or so (hex) digits of the ID. now it would print the full 64 characters of the hex ID which I think isn't necessary to identify the problematic beacon.

I think we can change `String()` to only print the first few characters or keep a `ShortString()` and use this in most places, especially for errors and fields in log messages.

Reply:
Beacon is 4 bytes long, which is 8 characters in hex. The `ShortString()` always printed the same string as `String()` (all 8 characters prefixed with "0x).

Reply:
OK then its fine 🙂